x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,52 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Coin,  // price to pay for each period, Coin contains both amount and denomination
```

Comment:
how would it work though? charge all of them or try charging one-by-one?

Reply:
Charge a portfolio of coins?

Reply:
It should charge all of them. It is consider as one single set of coins.

Reply:
not sure if there is any use case for that -- I'd favour the current simpler version

Reply:
@christopher-crypto may comment if it'd be useful to charge several currencies on a single subscription at a time?

Reply:
Yes, I think single Coin should work in most subscription use case, I also don't have sth in mind for mutliple coins at this moment.

The reason I suggest is I see other modules are doing it, so it is not hard to support (After all `Coins` is just an array of `Coin`, and the `Bank` keeper is supporting sending `Coins` instead of `Coin` actually (https://github.com/cosmos/cosmos-sdk/blob/master/x/bank/keeper/send.go#L19)

But I do agree because in fact on our current chain, all messages. e.g. `MsgSend`, we are always having only one token (`basetcro`) inside `Coins`.

Reply:
Since there's transfer for coins, then I think it's trivial to change it to `Coins`.

Reply:
done